.travis.yml

```diff
@@ -5,6 +5,10 @@ php:
   - 7.2
   - nightly
 
+matrix:
```

Comment:
This looks like it makes sense, but I don't want to introduce it if we aren't actually seeing a problem here. Could you tell me a bit more?

Reply:
The `php-nightly` is the unstable version in PHP.
And nobody can guarantee that the ```nightly``` build will be successful in every Travis CI build.
That's why I add this in the `allow_failures` in `.travis.yml` setting to avoid that happens.